DirectEmployers/MyJobs
# [PD-3611] Implement Confirm handling for TJD wizard navigation

## What's this PR do?
This PR does a few things

1) Expands the Confirm.jsx module to allow multi-response handling.
2) Adds a special case to clicking the "New TJD" navigation that verifies the user's intent
3) Makes resetting of the TJD wizard more explicit
4) Begins removing the "CBOModal" component
5) Moves wizard-specific files to a new wizard directory, instead of under the cbo directory

Bonus) Adds a "restartwebpack" dk command.. Cuz I got sick of doing app/stopapp when devserver alone failed.

## Where should the reviewer start?
TJD. Verify that the "New TJD" navigation opens a prompt when you've begun creating a new TJD, navigate away, and attempt to navigate back.

## How should this be manually tested?
1) Start a new TJD in the wizard (at LEAST enter a name and hit "Next")
2) Click another link, or press Back
3) Click "New TJD" again. You should be prompted with the new Confirm box.

## Any background context you want to provide?
This is the beginning of a refactor of the TJD wizard. Going forward, there will be tickets for
1) Detecting partial form inputs by either just putting the data in redux as users enter it, or prompting before they navigate away
2) Add history handling to the wizard (replacing the current switch statement method)
3) Remove uses of the CboModal component and replace with Confirm.
